kitsune/products/jinja2/products/product.html

```diff
@@ -11,15 +11,34 @@
 
 {% block title %}{{ _('{product} Help')|f(product=pgettext('DB: products.Product.title', product.title)) }}{% endblock %}
 
+{% block base_container_classes %}container_wide{% endblock %}
+
 {% block content %}
-  <div class="grid_12 search-form-large">
+<header class="help-topics-header">
+      <h1 class="product-title">
+        <img class="product-logo" src="{{ product.image_alternate_url }}" alt="" />
```

Comment:
Could we add an `alt` tag for this image? e.g. `Photo of {{ product.title }}`

Reply:
Sure – fixed.